src/workerd/io/worker-entrypoint.c++

```diff
@@ -257,20 +257,17 @@ kj::Promise<void> WorkerEntrypoint::request(
       // worker, not just for actors (and W2W below), but getting that right will require cleaning
       // up error handling more generally.
       return exceptionToPropagate();
-    } else KJ_IF_MAYBE(client, failOpenClient) {
+    } else KJ_IF_MAYBE(service, failOpenService) {
       // Fall back to origin.
 
       // We're catching the exception, but metrics should still indicate an exception.
       metrics->reportFailure(exception);
 
       auto promise = kj::evalNow([&] {
-        // kj::newHttpService adapts an HttpClient to look like an HttpService which makes it
-        // easier to forward the call.
-        auto httpWrapper = kj::newHttpService(**client);
-        auto promise = httpWrapper->request(
+        auto promise = service->get()->request(
```

Comment:
Just curious if `x->get()->f()` is preferred to `(*x)->f()`.  I assume they're equivalent.

Reply:
They are equivalent but I find it really awkward to type `(*foo)` because I always type `foo` first and then I'd have to move my cursor back to add the `*` and parentheses, whereas `->get()->f()` auto-completes.